### PR TITLE
Add option for FF connect notification.

### LINKED ISSUE #149: Option to disable rumble connection notification
**Is your feature request related to a problem? Please describe.**
Please provide option do disable rumble effect that notifies you about successful connection to the device.

**Describe the solution you'd like**
An additional option in configuration menu to disable rumble connection notification (as opposed to disabling rumble altogether) 

**Describe alternatives you've considered**
NA

**Additional context**
NA

### PR DESCRIPTION
**Pull request by adam-becker**
Closes #149

### CODE REVIEW

File: hid-xpadneo/src/hid-xpadneo.c [hunk 1]
```diff
@@ -57,13 +57,20 @@ static bool param_combined_z_axis;
 
 module_param_named(combined_z_axis, param_combined_z_axis, bool, 0644);
 MODULE_PARM_DESC(combined_z_axis,
-		 "(bool) Combine the triggers to form a single axis. 1: combine, 0: do not combine");
+		 "(bool) Combine the triggers to form a single axis. 1: combine, 0: do not combine.");
 
 static u8 param_trigger_rumble_damping = 0;
+
```
Comment (kakra) [inline hid-xpadneo/src/hid-xpadneo.c:63]: This empty line does not match the style of the block just above.

File: hid-xpadneo/src/hid-xpadneo.c [hunk 2]
```diff
@@ -57,13 +57,20 @@ static bool param_combined_z_axis;
 
 module_param_named(combined_z_axis, param_combined_z_axis, bool, 0644);
 MODULE_PARM_DESC(combined_z_axis,
-		 "(bool) Combine the triggers to form a single axis. 1: combine, 0: do not combine");
+		 "(bool) Combine the triggers to form a single axis. 1: combine, 0: do not combine.");
```
Comment (kakra) [inline hid-xpadneo/src/hid-xpadneo.c:60]: This change is unrelated and should be a separate commit, you may put it in the same PR, tho. The same for the change a few lines below.

File: hid-xpadneo/src/hid-xpadneo.c (hunk 1 above)
Comment (adam-becker) [reply hid-xpadneo/src/hid-xpadneo.c:63]: The style on this was already inconsistent before my PR, which one is preferred?

**Comment by adam-becker** [comment]
Also, does this need to be enabled by default?

File: hid-xpadneo/src/hid-xpadneo.c (hunk 1 above)
Comment (kakra) [reply hid-xpadneo/src/hid-xpadneo.c:63]: I think we should stick with how in-kernel drivers do it.

**Comment by kakra** [comment]
> Also, does this need to be enabled by default?

The notification should play by default so we can easily catch clones not supporting the rumble protocol correctly.

File: hid-xpadneo/src/hid-xpadneo.c (hunk 1 above)
Comment (adam-becker) [reply hid-xpadneo/src/hid-xpadneo.c:63]: I'm not sure what that looks like. The code in `master` doesn't follow a consistent style either.

Comment (kakra) [reply hid-xpadneo/src/hid-xpadneo.c:63]: I'll take a peek when I get to work on the code next time and fix this properly.

File: hid-xpadneo/src/hid-xpadneo.c [hunk 3]
```diff
@@ -57,13 +57,20 @@ static bool param_combined_z_axis;
...
 
 static u8 param_trigger_rumble_damping = 0;
+
 module_param_named(trigger_rumble_damping, param_trigger_rumble_damping, byte,
 		   0644);
 MODULE_PARM_DESC(trigger_rumble_damping,
-		 "(u8) Damp the trigger: 1 (none) to 2^8+ (max)");
+		 "(u8) Damp the trigger: 1 (none) to 2^8+ (max).");
```
Comment (kakra) [inline hid-xpadneo/src/hid-xpadneo.c:67]: I'm not sure if this is correct anyways. I never closely looked at the damping code. But when anything, the range for u8 is probably 0..255 and not 1..256 (or whatever `2^8+` should mean).

Comment (adam-becker) [reply hid-xpadneo/src/hid-xpadneo.c:67]: I'll leave that work for a future PR, just wanted the descriptions to look the same.

Comment (adam-becker) [reply hid-xpadneo/src/hid-xpadneo.c:67]: Also, it looks like it has an effective range of `1..255`
